.github/actions/verify-token/action.yml

```diff
@@ -31,7 +31,11 @@ inputs:
     default: ${{ github.token }}
 
   output-predicate:
-    description: "The filename to place the output SLSA predicate based on the verified token"
+    description: "The filename to place the output SLSA predicate based on the verified token."
+    required: true
+
+  builder-interface-type:
```

Comment:
Could the default just be "builder"?

Reply:
we could, but I thought making it explicit was better. Wdut?